src/main/java/com/google/impactdashboard/server/DataUpdater.java

```diff
@@ -38,13 +39,36 @@ public static DataUpdater create() throws IOException {
   /**
    * Updates the database with any new information about recommendations and IAMBinding logging.
    */
-  public void updateDatabase() {}
+  public void updateDatabase() {
+    updateManager.deleteYearOldData();
+    updateManager.updateRecommendations(listUpdatedRecommendations());
+    updateManager.updateIAMBindings(listUpdatedIAMBindingData());
+  }
```

Comment:
Could you maybe swap the order of these operations? The database backend deletes data that is 365 days older than the newest data it has, so it would be helpful to already have the newest data when it's trying to figure out which data is 365 days old.

Reply:
Yeah I can do that